doc/dev.md

```diff
@@ -213,18 +213,23 @@ xcodebuild -downloadPlatform iOS
 export PATH="$(brew --prefix flex)/bin:$(brew --prefix bison)/bin:$PATH"
 
 # Configure using CMake
+#
+# Make sure to update the `VCPKG_HOST_TRIPLET` parameter below if you are using an x64 MacOS system to the value `x64-osx`. There are
+# also known issues on MacOS, check this issue for the current status: https://github.com/opengisch/QField/issues/5897.
 
 cmake -S . -B build-arm64-ios \
+   -DVCPKG_HOST_TRIPLET=arm64-osx
```

Comment:
Spacing issue here (as well as below with DCMAKE_SYSTEM_PROCESSOR , can it be fixed?

Reply:
Updated